### DISCUSSION TITLE
Using Github Projects section for manage srsRAN community feature requests/roadmap

### DISCUSSION THREAD

**Discussion by ohault** [srsran/srsRAN_Project#962] [Ideas & Suggestions]
As feature requests input are not accepted as Github issues in this project, I would suggest to use the GitHub Project feature to manage community feature requests/roadmap, in place of the current forum based Discussions feature.

**Comment by pgawlowicz** [comment]
@ohault Thank you for moving the post here. 
One minor thing: could you correct the project name in the thread title? It should be srsRAN, not srsRAM.

**Reply by ohault** [reply]
> @ohault Thank you for moving the post here. One minor thing: could you correct the project name in the thread title? It should be srsRAN, not srsRAM.

Of course, thank you.
